library/Icinga/Application/Hook/MigrationHook.php

```diff
@@ -361,8 +362,8 @@ protected function loadLastState(string $version): ?string
         /** @var array<string, string> $states */
         $states = $session->get($this->getModuleName(), []);
         if (! isset($states[$version])) {
-            $schemaQuery = $this->getSchemaQueryFor($version);
-            $schemaQuery->setFilter(Filter::all(Filter::equal('success', 'n')));
+            $schemaQuery = $this->getSchemaQuery()
+                ->setFilter(Filter::all(Filter::equal('success', 'n')));
```

Comment:
The version doesn't need to be filtered anymore?

Reply:
Previous version of this code:
```php
$schemaQuery = $this->getSchemaQueryFor($version);
$schemaQuery->setFilter(Filter::all(Filter::equal('success', 'n')));
```

The version filter was never used `Query::setFilter()` overrides it.